codecov.yml

```diff
@@ -0,0 +1,9 @@
+coverage:
+  range: "60...100"
```

Comment:
It is impractical to reach 100% test coverage, I think 95 is a better upper limit.